unstructured_ingest/v2/processes/connectors/onedrive.py

```diff
@@ -101,7 +101,7 @@ def precheck(self) -> None:
 
     def list_objects(self, folder, recursive) -> list["DriveItem"]:
         drive_items = folder.children.get().execute_query()
-        files = [d for d in drive_items if d.is_file]
+        files = [d for d in drive_items if d.file is not None]
```

Comment:
Changed to match [this example](https://github.com/vgrem/Office365-REST-Python-Client?tab=readme-ov-file#example-download-the-contents-of-a-driveitemfolder-facet)